### ISSUE TITLE
Importing SQL rowfile module not found

### ISSUE THREAD

**Issue by valtido** [Azure/azure-functions-core-tools#2343]
I have a simple issue when running `func` or `npm run start`.

in package.json I have 
```
....
start: "func",
...
```

on my project file I have 
```
...
import sqlInsert from 'foo.sql'; // this is a simple sql file
,,,
```

```
//foo.sql
SELECT * from Table;
```

When I `npm run start` it shows the following error.
`Exception: Worker was unable to load function GetDataProviderResponse: 'Error: Cannot find module 'foo.sql'`

**Comment by apawast** [comment]
@mhoeger could you please take a look at this?

**Comment by anthonychu** [comment]
@valtido I don't believe `import sqlInsert from 'foo.sql'` is expected to work. Node only knows how to import JavaScript files. To read the SQL file, try using [`fs.readFileSync()`](https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fs_readfilesync_path_options).

**Comment by valtido** [comment]
It is in typescript, which eventually should be good.

I also used the readFileSync, but eventually the file doesn't end up on the dist folder.

alternatively is there a ts version of "func" I can run it on?
